zanata-war/src/main/webapp-jboss/WEB-INF/classes/META-INF/persistence.xml

```diff
@@ -131,7 +133,9 @@
            See http://stackoverflow.com/a/8701914
       -->
       <property name="hibernate.cache.region.factory_class"
-        value="org.hibernate.cache.ehcache.SingletonEhCacheRegionFactory" />
+        value="org.hibernate.cache.infinispan.JndiInfinispanRegionFactory" />
+      <property name="hibernate.cache.infinispan.cachemanager"
+        value="java:jboss/infinispan/container/hibernate" />
```

Comment:
@seanf This is where you specify those things.

But if we're thinking about providing Zanata's own infinispan, then this setup is shared by all versions and doesn't present a problem at all. Not sure if we would still use JNDI for getting a hold of it though.